packages/exportable-manifest/src/index.ts

```diff
@@ -57,6 +57,10 @@ export default async function makePublishManifest (dir: string, originalManifest
       })
   }
 
+  if (readmeFile) {
+    publishManifest.readme = readmeFile
```

Comment:
May be `publishManifest.readme ??= readmeFile` ? How npm behaves in similar case?

Reply:
Yes, you're right, it would be corrrect https://github.com/npm/cli/blob/d8aac8448e983692cacb427e03f4688cd1b62e30/node_modules/read-package-json/read-json.js#L291